builtin/credential/approle/path_role.go

```diff
@@ -116,104 +105,90 @@ type roleIDStorageEntry struct {
 // role/<role_name>/secret-id-accessor/lookup - For reading secret_id using accessor
 // role/<role_name>/secret-id-accessor/destroy - For deleting secret_id using accessor
 func rolePaths(b *backend) []*framework.Path {
-	return []*framework.Path{
-		&framework.Path{
-			Pattern: "role/?",
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ListOperation: b.pathRoleList,
+	defTokenFields := tokenutil.TokenFields()
```

Comment:
https://www.vaultproject.io/api/auth/approle/index.html#read-update-or-delete-approle-properties will not be comprehensive anymore. I am not sure what the right path here. Should we add individual paths for all the newer fields?

Reply:
No, not unless someone complains/needs it.